packages/website/src/components/AddEthereumChainButton/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+
+type Props = {
+  buttonText: string;
+  chain: string;
+};
+
+async function addEthereumChain(chain: string) {
+  interface AddEthereumChainParameter {
+    chainId: string; // A 0x-prefixed hexadecimal string
+    chainName: string;
+    nativeCurrency: {
+      name: string;
+      symbol: string; // 2-6 characters long
+      decimals: 18;
+    };
+    rpcUrls: string[];
+    blockExplorerUrls?: string[];
+    iconUrls?: string[]; // Currently ignored.
+  }
+  const l1params: AddEthereumChainParameter = {
+    chainId: "0x7A68",
+    chainName: "Taiko Testnet L1",
```

Comment:
I added the L1/L2 networks to metamask in bridge, seems the bridge uses different names?
![image](https://user-images.githubusercontent.com/104078303/209599210-0e13119d-542a-456d-b517-6fed6b886618.png)

Reply:
if you remove it and try to add from bridge again, it will be Taiko A1. i updated it.